src/libs/ReportUtils.ts

```diff
@@ -2161,11 +2161,11 @@ function getDisplayNameForParticipant(
 
     // If the user's personal details (first name) should be hidden, make sure we return "hidden" instead of the short name
     if (shouldFallbackToHidden && longName === hiddenTranslation) {
-        return longName;
+        return LocalePhoneNumber.formatPhoneNumber(longName);
     }
 
     const shortName = personalDetails.firstName ? personalDetails.firstName : longName;
-    return shouldUseShortForm ? shortName : longName;
+    return LocalePhoneNumber.formatPhoneNumber(shouldUseShortForm ? shortName : longName);
```

Comment:
@Kalydosos I am trying to test this change with various users those have phone numbers, but I’m unsure why we need LocalePhoneNumber.formatPhoneNumber here. Each time I log and check, both shortName and longName are already formatted (no SMS and no country code prefix). 

Could you let me know if there’s a way to test this specific change?

Reply:
yes they are already formatted at that line
![Capture d’écran de 2024-12-06 15-56-26](https://github.com/user-attachments/assets/7692d423-585d-4d43-b4ca-4bef6b30b2be)